Modules/Projects/Feature/Modules/TasksExportImportTest.php

```
@@ -0,0 +1,205 @@
+        foreach ($tasks as $task) {
+            $this->assertStringContainsString($task->task_name, $content);
+        }
+    }
+
+    #[Test]
+    #[Group('export')]
+    public function it_exports_tasks_downloads_excel_with_correct_data(): void
+    {
+        $this->markTestIncomplete();
+        /* Arrange */
+        $tasks = Task::factory()->for($this->company)->count(3)->create();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$tasks'. (undefined)

(UnusedLocalVariable)